otx/cli/tools/train.py

```diff
@@ -69,14 +69,19 @@ def parse_args():
     required = not os.path.exists(parsed.data)
 
     parser.add_argument(
-        "--train-data-roots",
+        "--data-root",
```

Comment:
With this structure (data-root + subset's ann-file), the folder structure format of multi-class classification is not considered. What do you think?

Reply:
hmm, no problem. I'll check if I can support folder structure in multi-class classification